src/sections/About/About.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import './About.css';
+import UserProfile from '../../assets/images/UserProfile.png'
+import AppStoreButtons from '../../assets/AppStoreButtons/AppStoreButtons';
```

Comment:
The buttons were commented out at the bottom but not at the top. Besides that everything is perfect.